sympy/core/tests/test_random.py

```diff
@@ -0,0 +1,55 @@
+import random
+from sympy import Symbol, symbols, sin, acos
+from sympy.abc import x as xx
+
+
+# Values should be equal:
+random.seed(42)
+before = random.random()
+
+random.seed(42)
+Symbol('z').is_finite
+after = random.random()
+assert before == after
```

Comment:
Tests should be in functions called `test_something` rather than top-level in the module